hack/make/lint.mk

```diff
@@ -19,5 +19,5 @@ else
 CHECKSTYLE :=
 endif
 
-GOMETALINTER := gometalinter --vendor --tests --vendored-linters $(CHECKSTYLE) --disable=gotype --enable=unused --enable=goimports --enable=gofmt --concurrency=$(NPROCS) --deadline=600s --exclude=api/netflix/titus --exclude=/usr/local/go/src
+GOMETALINTER := gometalinter --vendor --tests --vendored-linters $(CHECKSTYLE) --disable=gotype --enable=unused --enable=goimports --enable=gofmt --concurrency=$(NPROCS) --deadline=600s --exclude=executor/runtime/docker/seccomp --exclude=api/netflix/titus --exclude=/usr/local/go/src
```

Comment:
maybe move the non-lintable pieces to a separate file and exclude it, instead of the whole package?

Reply:
The (entire) package is generated. Let me add some stuff about it there.

Reply:
Mixing generated and non-generated code in a single package seems to end poorly. Mostly because the generator internally relies on some private methods, and variables that are "unstable" API-wise. Mixing code in the same package opens you up to accidentally foot shooting.

Reply:
OK, I didn't know the entire package was generated. If that is the case, :+1: to ignoring it all.